docs/README.md

```diff
@@ -0,0 +1,277 @@
+# Microsoft Azure SDK for Ruby
+
+As of (2022/01), Azure SDK for Ruby - Resource Management Libraries are officially deprecated. The libraries will be in maintenance mode for a period of 3 years and security / bug fixes will still be applied. However, there will be no further releases based on feature requests. 
+
+We recommend you take one of the following approaches
+- Use Azure REST API directly in Ruby (instructions are provided in this doc)
+- Use one of the next-generation SDKs that provides more idiomatic, consistent, approachable, dependable, and diagnosable experience. You can find more info on the repo of each individual language
```

Comment:
No we can't say that. If customers are using Ruby, we can't tell them to switch language

Reply:
> No we can't say that. If customers are using Ruby, we can't tell them to switch language

OK. I will remove this paragraph.